server/src/main.rs

```diff
@@ -136,71 +136,78 @@ async fn main() -> Result<(), Error> {
 
     #[cfg(debug_assertions)]
     let backend = {
-        use rand::prelude::SliceRandom;
-        use serde::Deserialize;
-        use std::time::Duration;
+        // To be able to develop and test against a local DynamoDB instance
+        // USE_DYNAMODB=1 AWS_ENDPOINT_URL=http://localhost:8000 cargo run
+        if std::env::var_os("USE_DYNAMODB").is_some() {
+            let config = aws_config::load_from_env().await;
+            Backend::Dynamo(aws_sdk_dynamodb::Client::new(&config))
+        } else {
```

Comment:
I'd rather we just change the whole thing to:
```rust
let backend = if !cfg!(debug_assertions) || std::env::var_os("USE_DYNAMODB").is_some() {
    let config = aws_config::load_from_env().await;
    Backend::Dynamo(aws_sdk_dynamodb::Client::new(&config))
} else {
    // what's there now
};

Reply:
Refactored. Also you might be wondering why we are not seeding in this arm - I wanted the seeding to a local instance to be in a dedicated PR to first make sure we are conceptually ok :)